.github/workflows/deploy-to-devnet.yml

```diff
@@ -1,5 +1,11 @@
 name: Deploy to Devnet
 
+# This workflow performs automatic deployment of aleph-node to the Devnet environment
+# It does it from the scratch, ie it
+#   1) syncs the validators keys from S3,
+#   2) generates raw chainspec from the deployed aleph-node binary,
+#   3) restart nodes with cleaned db
+
```

Comment:
Can we change all the set-output calls in this workflow, like mentioned on this post: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/ ?

Reply:
done